apiserver/facades/client/modelupgrader/upgrader.go

```diff
@@ -236,6 +237,7 @@ func (m *ModelUpgraderAPI) UpgradeModel(arg params.UpgradeModelParams) (result p
 func preCheckEnvironForUpgradeModel(
 	ctx context.ProviderCallContext, env environs.BootstrapEnviron,
 	controllerModel bool, currentVersion, targetVersion version.Number,
+	logger loggo.Logger,
```

Comment:
This function even has "ForUpgradeModel" in the name, why tf is it not a method?